kchd/controllers/system_status.py

```diff
@@ -86,19 +85,14 @@ def astoria_good(self) -> bool:
         """Determine whether astoria services are in a good state."""
         return len(self._seen_services) == len(self._required_services)
 
-    def get_enabled_leds(self) -> int:
+    def get_enabled_leds(self) -> Tuple[bool, bool, bool]:
```

Comment:
The object returned from this function is not a tuple.

```suggestion
    def get_enabled_leds(self) -> List[bool]:
```

Reply:
This was a mistake: a636fc5